XcodeServerSDK/XcodeServerConfig.swift

```diff
@@ -33,16 +42,51 @@ public class XcodeServerConfig : JSONSerializable {
         return dict
     }
     
-    public init(var host: String, user: String?, password: String?) {
+    ///
+    /// Initializes a server configuration with the provided host.
```

Comment:
How are you generating these headers? Do we not want to use the classic javadoc format with `/**`?